src/app/shared/enum/applications.ts

```diff
@@ -13,4 +13,13 @@ export enum ApplicationTitles {
   'Зараховано',
   'Відмовлено',
   'Гурток залишено',
+  'Заблоковано'
+}
+export enum ApplicationStatusDescription {
+  'Усі',
```

Comment:
lets discuss  how we can store all titles and descriptions to support multi languages